addons/hr_holidays/report/hr_leave_reports.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0"?>
+<odoo>
+        <record id="view_holiday_pivot" model="ir.ui.view">
+            <field name="name">hr.holidays.report_pivot</field>
+            <field name="model">hr.leave</field>
```

Comment:
hr.leave views should stay in hr_leave_views file. Those are not really report view, only classic views of pivot or graph type.